frontend/stories/inputs/chip/Chip.stories.tsx

```diff
@@ -56,15 +55,18 @@ export default {
 				type: 'select',
 			},
 		},
+		onDelete: {
+			defaultValue: null,
+		},
 	},
-	parameters: { controls: { exclude: ['size', 'onDelete', 'deleteIcon', 'avatar', 'sx', 'classes', 'clickable', 'children'] } },
+	parameters: { controls: { exclude: ['size', 'onDelete', 'deleteIcon', 'avatar', 'sx', 'classes', 'clickable', 'children', 'ref'] } },
 } as ComponentMeta<typeof Chip>;
 
 // More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
 const Template: ComponentStory<typeof Chip> = (args) => (
```

Comment:
I think you can re-write this simply as `const Template: ComponentStory<typeof Chip> = Chip;`